moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -257,21 +257,19 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
    * @param {number} statusId - Project's status id
    * @return {JSX.Element}
    */
-  const buildStatusBadge = (phase, statusId) => (
+  const buildStatusBadge = (phase) => (
     <ProjectStatusBadge
-      status={statusId}
```

Comment:
#872 completely reworks the status badge component. Here, the effect of removing the `status` prop is that the status badge will not render if the project is missing a current phase.